daily_reports.rb

```diff
@@ -36,6 +36,7 @@ def all_projects(date, slack, text, rerun, verbose, customer_facing, short)
       error = <<~MSG
       Generation of daily report for project *#{project.name}* stopped due to error:
       #{e}
+      #{e.error_messages}
```

Comment:
Would it be too cheeky to add an `error_messages` instance variable to `AwsSdkError` too? It would never be used, and this block would just print an empty array, but I think it's a better idea than using an Azure-specific conditional in the error printing.

Reply:
Yeah that sounds fair enough.